docs/vlab/demo.md

```diff
@@ -86,14 +86,126 @@ graph TD
+     default route from external permitted
+   1~as5835:subnets=default,other:prefixes=0.0.0.0/0_le32_ge32,22.22.22.0/24 -- same but with more details
+   1~as5835:s=default,other:p=0.0.0.0/0_le32_ge32,22.22.22.0/24 -- same as above
+
+OPTIONS:
+   --help, -h                     show help
+   --name value, -n value         name of the VM or HW to access
+   --wait-switches-ready, --wait  wait for switches to be ready before before and after configuring peerings (default: true)
+
+   Global options:
+
+   --brief, -b      brief output (only warn and error) (default: false) [$HHFAB_BRIEF]
+   --cache-dir DIR  use cache dir DIR for caching downloaded files (default: "/home/ubuntu/.hhfab-cache") [$HHFAB_CACHE_DIR]
+   --verbose, -v    verbose output (includes debug) (default: false) [$HHFAB_VERBOSE]
+   --workdir PATH   run as if hhfab was started in PATH instead of the current working directory (default: "/home/ubuntu") [$HHFAB_WORK_DIR]
+```
+
+### Test Connectivity
+`hhfab vlab` includes a utility to test connectivity between servers inside VLAB. This utility is a `hhfab vlab` sub-command. `hhfab vlab test-connectivity`.
+
+```
+NAME:
+   hhfab vlab test-connectivity - test connectivity between all servers
+
+USAGE:
+   hhfab vlab test-connectivity [command options]
+
+OPTIONS:
+   --curls value                  number of curl tests to run for each server to test external connectivity (0 to disable) (default: 3)
+   --help, -h                     show help
+   --iperfs value                 seconds of iperf3 test to run between each pair of reachable servers (0 to disable) (default: 10)
+   --iperfs-speed value           minimum speed in Mbits/s for iperf3 test to consider successful (0 to not check speeds) (default: 7000)
+   --name value, -n value         name of the VM or HW to access
+   --pings value                  number of pings to send between each pair of servers (0 to disable) (default: 5)
+   --wait-switches-ready, --wait  wait for switches to be ready before testing connectivity (default: true)
+
+   Global options:
+
+   --brief, -b      brief output (only warn and error) (default: false) [$HHFAB_BRIEF]
+   --cache-dir DIR  use cache dir DIR for caching downloaded files (default: "/home/ubuntu/.hhfab-cache") [$HHFAB_CACHE_DIR]
+   --verbose, -v    verbose output (includes debug) (default: false) [$HHFAB_VERBOSE]
+   --workdir PATH   run as if hhfab was started in PATH instead of the current working directory (default: "/home/ubuntu") [$HHFAB_WORK_DIR]
+
+```
 ## Manual VPC creation
 ### Creating and attaching VPCs
 
 You can create and attach VPCs to the VMs using the `kubectl fabric vpc` command on the Control Node or outside of the
 cluster using the kubeconfig. For example, run the following commands to create 2 VPCs with a single subnet each, a DHCP
 server enabled with its optional IP address range start defined, and to attach them to some of the test servers:
 
-```console
+```
```

Comment:
It doesn't matter much here - these blocks _are_ `console` output, because they contain both commands and their output. However, we won't get the highlight for commands, because the `core@control-1` prefix in front of the prompt symbol and commands prevents GitHub to recognise the commands.

Reply:
So I have been looking at the docs as rendered via `make serve` I did that mass removal to avoid numbers in the output being colored for reasons I couldn't easily deduce. For examples the first 10 in the ip addresses. The rendering in GitHub is something that I have only started looking at because of this comment. Are you okay with a policy of, no console tag at all since the rendering seems to be spotty be between material-mkdocs and github?

Reply:
Yes I'm fine :) Apologies if my advice was misleading, I'm used to GitHub's rendering but it's true I've little experience myself with mkdocs, so maybe my recommendations were not for the best. Do use what renders best in your tests :+1:

Reply:
Will do. Also I am looking to update material-mkdocs but I have noticed some differences between the versions.